src/gui/sdlmain.cpp

```diff
@@ -195,6 +194,19 @@ extern char** environ;
 
 SDL_bool mouse_is_captured = SDL_FALSE; // global for mapper
 
+// masks to be passed when creating SDL_Surface
+#ifndef MACOSX
+#if SDL_BYTEORDER == SDL_BIG_ENDIAN
+constexpr uint32_t RMASK = 0xff000000;
+constexpr uint32_t GMASK = 0x00ff0000;
+constexpr uint32_t BMASK = 0x0000ff00;
```

Comment:
This should cover the big-endian PPC non-macOS systems like @kas1e and I have. 
For macOS systems running big-endian processors, like the G5 - do they use little-endian when it comes to SDL?

Reply:
I think it will be a bit clearer when looking at context of this change: 0b87c0bc6a0a88a33ea6dbefa2b073faeddeda48

In old code, these masks were used in 2 contexts:
1. For rendering icon used as a fallback when icon is not set other way. This code was ifdefed out for macOS, with an explanation that AppBundle includes nicer icon, so this one is not needed… This explanation is only partly true, but *shrug* - I decided to preserve the behaviour.  *Side note: this code was also broken in SVN for who-knows-how-long, and the fallback icon was not showing up on Linux - but now it does :)*
2. For rendering warning messages in a window (on Windows only) - I have never ever seen this code in action on any Windows build, but it's there. Not sure why PPC ifdefs were appearing in Windows-specific code (maybe it was used on OS/2 as well in the past?) No idea.

So, after moving these masks out of function scopes, our CI hit me with a new warning: unused variable, only on macOS builds.